smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/lang/KotlinTypes.kt

```diff
@@ -34,9 +34,14 @@ object KotlinTypes {
     val Double: Symbol = builtInSymbol("Double")
     val Boolean: Symbol = builtInSymbol("Boolean")
 
-    val List: Symbol = builtInSymbol("List", "kotlin.collections")
-    val Set: Symbol = builtInSymbol("Set", "kotlin.collections")
-    val Map: Symbol = builtInSymbol("Map", "kotlin.collections")
+    object Collections {
+        val List: Symbol = builtInSymbol("List", "kotlin.collections")
+        val Set: Symbol = builtInSymbol("Set", "kotlin.collections")
+        val Map: Symbol = builtInSymbol("Map", "kotlin.collections")
+    }
+    object Text {
+        val encodeToByeArraySymbol = builtInSymbol("encodeToByteArray", "kotlin.text")
```

Comment:
**Nit**: `Bye` → `Byte`

Also, we don't typically suffix `Symbol` to these definitions. Everything in this file should be symbols so explicitly calling them such is not super-helpful.

Reply:
nit

we don't add `Symbol` suffix elsewhere and could just be `encodeToByteArray`. This is just carryover from relocation I think.